modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchContextAttributes.java

```diff
@@ -29,6 +29,6 @@ public class SearchContextAttributes {
 		"search.lucene.syntax";
 
 	public static final String ATTRIBUTE_PERMISSIONED_SEARCHER =
```

Comment:
@felipelorenz - could you change ATTRIBUTE_PERMISSIONED_SEARCHER to ATTRIBUTE_KEY_PERMISSIONED_SEARCHER to match